helpers/file-upload.helper.js

```diff
@@ -0,0 +1,23 @@
+const multer = require('multer');
+const fileStorage = multer.diskStorage({
+  // Destination to store file
+  destination: 'uploads',
+  filename: (req, file, cb) => {
+    cb(null, file.originalname);
+  }
+});
+
+const fileUpload = multer({
+  storage: fileStorage,
+  fileFilter(req, file, cb) {
```

Comment:
use appropriate name for callback function

Reply:
made the required changes